ui/src/components/Multi.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from "react";
+import { MultiSelect } from "react-multi-select-component";
+
+const handleNewField = (value) => ({
+  label: value,
+  value: value.toUpperCase(),
```

Comment:
the label and value's value can probably just be value here

Reply:
i think the label is built in so that it is read instead of value to show up on the ui. whenever i get rid of the labels it breaks it. maybe we could take a look at this.

Reply:
```
{
label: name,
value: name
}
```